### ISSUE TITLE
Allow the version of a DMN decision table to be specified in a model, and use that version

### ISSUE THREAD

**Issue by jwulf** [camunda/camunda#9903]
**Is your feature request related to a problem? Please describe.**

When I deploy a DMN decision table on Camunda 8, I get back a version.

However, a Business Rule task does not allow me to specify a version of the decision table to use.

This means using the DMN is not a safe approach for long running process as it will always pick latest version and any DMN changes will impact already running instances state.

**Describe the solution you'd like**

That I can specify in the model the version of the decision table to use, and it will be respected by the engine. 

**Describe alternatives you've considered**
Not using DMN.

**Additional context**
I have opened an issue in the Modeler repo for the model: https://github.com/camunda/camunda-modeler/issues/3054

Product Hub Epic: https://github.com/camunda/product-hub/issues/349

**Comment by menski** [comment]
@felix-mueller are you aware of this feature requests, is there already a product hub epic?

**Comment by felix-mueller** [comment]
Yes PH-iSsue 349  https://github.com/camunda/product-hub/issues/349
